testbed/testbed.sh

```diff
@@ -45,7 +45,7 @@ testcase () {
 
     # Detect takeoff on a distance from origin >= 0.5 meters
     # Fail test if no takeoff is detect after 5 seconds
-    sleep_until_takeoff 0.5
+    sleep_until_takeoff 0.5 10
```

Comment:
Its working here. It will only count the time after coav-sim.sh is already up (sleep 30 before that) so it looks to me like enough time for the drone to takeoff.